bit_manipulation/binary_or_operator.py

```diff
@@ -0,0 +1,51 @@
+"""
+https://www.tutorialspoint.com/python3/bitwise_operators_example.htm
+This function takes in 2 integer, convert them to binary and returns a binary
+in str format that is the result of an Binary OR operation from the 2 integer
+input.
+
+Returns a binary resulted from 2 integer input in str format
+>>> binary_or(25,32)
+'0b111001'
+
+>>> binary_or(37,50)
+'0b110111'
+
+>>> binary_or(21,30)
+'0b11111'
+
+>>> binary_or(58,73)
+'0b1111011'
+"""
+
+
+def binary_or(a : int, b : int):
+    if isinstance(a, float) or isinstance(b, float):
+        raise TypeError("'Float' object cannot be implemented as an integer")
+    if isinstance(a, str) or isinstance(b, str):
+        raise TypeError("'str' object cannot be implemented as an integer")
+    if a < 0 or b < 0:
+        raise ValueError("the value of both input must be positive")
+    """
+    a_binary and b_binary are the binary of a and b in str format
+    """
+    a_binary = str(bin(a))[2:]
+    b_binary = str(bin(b))[2:]
+    binary = []
+    if len(a_binary) > len(b_binary):
+        greater = len(a_binary)
+        b_binary = b_binary.zfill(greater)
+    else:
+        greater = len(b_binary)
+        a_binary = a_binary.zfill(greater)
+    for i in range(greater):

```

Comment:
Nice!  So how could we use `zip()` https://docs.python.org/3/library/functions.html#zip
```python
    for char_a, char_b in zip(a_binary, b_binary):
```

Reply:
probably just swapping out the for loop for that 
for char_a, char_b in zip(a_binary, b_binary):
        if char_a == "1" or char_b == "1":
            binary.append("1")
        else:
            binary.append("0")
I'm not entirely sure if there is a simpler way for it or not